app/pages/admin.js

```diff
@@ -91,25 +100,21 @@ function Admin() {
       });
   };
 
+  async function onCheckChange(org) {
+    org.isHidden = !org.isHidden;
+    updateOrg(org);
+  }
+
   const contentWhitelist = (
     <article className="card">
-      <h2>Manage Whitelist</h2>
+      <h2>Manage Whitelist (Deprecated)</h2>
       <p>
         Update the whitelist with this virtual text file. Each line must be a
         valid email or blank (these get stripped out) to submit.
       </p>
       <div className={whitelistHeader}>
         <code>whitelist.txt</code>{' '}
         {lastUpdated && <span>Last Updated: {lastUpdated}</span>}
-        {/* <label htmlFor="user-search">
```

Comment:
Unrelated to this line, but maybe we could add a search for orgs (just a caseless substring search for some metastring like `user's full name + org name`)

Reply:
This can be done in a subsequent PR. Just want to get this ability in soon.